### DISCUSSION TITLE
InputGroup and InputGroupAddon appear to be missing a module on V17

### DISCUSSION THREAD

**Discussion by Joe-EdgeTech** [primefaces/community#692] [PrimeNG]
I can't seem to find any module to import that adds the `InputGroup` or `InputGroupAddon` components.

I've tried a few modules like `InputTextModule` and `InputNumberModule` but I continually get 
```'p-inputGroup' is not a known element:```

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue? It looks normal, cannot replicate the error in here https://stackblitz.com/edit/qk2h61?file=package.json

**Comment by designrwells** [comment]
I discovered this as well. I noticed when comparing [the documentation](https://primeng.org/inputgroup) against the StackBlitz, the documentation shows `import { InputGroupAddon } from 'primeng/inputgroupaddon';` when it should be **InputGroupAddonModule**.

Hope that helps!
